.github/workflows/primer_run_pr.yaml

```diff
@@ -5,6 +5,9 @@
 name: Primer / Run
 
 on:
+  branches:
+    - main
+    - "maintenance/**"
```

Comment:
Like I said, I don't think we should ignore here.

Also doesn't this actually run this on `main` now instead of not running it?

Reply:
I'm not trying to ignore this is ``branches`` not ``branches-ignore``. If I understand well this is something that should be run only on the PR branch and we should ``branches-ignore`` both main and maintenance/** ?